src/components/DeveloperDocsLinks.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Header3, ListItem } from "./SharedStyledComponents"
+import Translation from "./Translation"
+import Link from "./Link"
+import docLinks from "../data/developer-docs-links.yaml"
+
+const DeveloperDocsLinks = () => {
+  return docLinks.map((items) => (
+    <>
+      <Header3>
+        <Translation id={items.id} />
+      </Header3>
+      <ul>
+        {items.items &&
+          items.items.map((item) => {
+            return (
+              <ListItem>
+                <Link to={item.to || item.path}>
+                  <Translation id={item.id} />
+                </Link>
+                <em>
+                  {" – "}
+                  <Translation id={item.descp_id} />
+                </em>
```

Comment:
[`em` for emphasis, `<i>` for italics](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em).

I think `<i>` is correct here.

```suggestion
                <i>
                  {" – "}
                  <Translation id={item.descp_id} />
                </i>
```

Reply:
You are right. `<i>` makes more sense. I used `<em>` because that's what is currently rendered on the website.
<img src="https://user-images.githubusercontent.com/4337699/131474743-e9dd6cb4-343b-4ada-83a1-f063d53152bd.png" width="400px" />

Reply:
I have now changed it to `<i>`

Reply:
Yeah, our markdown parser automatically uses `<em>` for anything wrapped in single asterisks or underscores.  In many circumstances this ends up being technically incorrect from an accessibility standpoint (for anyone wondering the difference, `<i>` is for visual styling while `<em>` is for _emphasis_ being placed on the word and is useful for screen readers to help read content more naturally).

Agreed in this case that `<i>` is more appropriate.